src/mono/wasm/debugger/tests/debugger-test/debugger-test.csproj

```diff
@@ -7,6 +7,10 @@
     <WasmBuildAppDependsOn>PrepareForWasmBuildApp;$(WasmBuildAppDependsOn)</WasmBuildAppDependsOn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(RuntimeConfiguration)'=='Debug'">
```

Comment:
The property changes in this file would be useful for other test assemblies too. They can be moved to `tests/Directory.Build.props`.